README.md

```diff
@@ -1,2 +1,49 @@
-# first-capstone-project
-this is the first capstone project for the first module in microverse
+
```

Comment:
- [ ] Kindly add a presentation video as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Once%20you%20have,very%20proud%20of).

Reply:
done